extra-utils/genie/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=genie
+PKGSEC=utils
+PKGDEP="daemonize dbus dotnet-runtime-3.1 dotnet-sdk-3.1 systemd"
+PKGDES="Fake systemd for WSL 2"
```

Comment:
"Fake systemd for use with WSL 2" might work better.

Reply:
Copied the explanation from upstream